### DISCUSSION TITLE
Incorporate into a form without Livewire

### DISCUSSION THREAD

**Discussion by kai0310** [filamentphp/filament#3135] [Help] [answered]
Only file uploaders use this package and incorporate it into regular forms that do not use Livewire.
I found a way to pass the path of the uploaded image file to the regular form as follows
However, this code would load the image forever. Perhaps I should not use `getState()` in the blade.

Is there any other way to get the contents of the form?

```PHP
<div>
    <input
        type="hidden"
        name="uploadedImage"
        id="uploadedImage"
        value="{{ $this->form->getState()['image'] }}"
    />
    {{ $this->form }}
</div>

```

[![Image from Gyazo](https://i.gyazo.com/50ec6b5c2c8ccbe31b17e2fcd482762f.gif)](https://gyazo.com/50ec6b5c2c8ccbe31b17e2fcd482762f)

Thanks.

**Comment by danharrin** [comment, marked as answer]
Sorry, we don't support this use case. The package was not designed for that.
